test/common/README.md

```diff
@@ -431,6 +432,48 @@ Decrements the `Countdown` counter.
 Specifies the remaining number of times `Countdown.prototype.dec()` must be
 called before the callback is invoked.
 
+## CPU Profiler module
+
+The `cpu-prof` module provides utilities reelated to CPU profiling tests.
+
+### env
+
+* Default: { ...process.env, FIB, NODE_DEBUG_NATIVE: 'INSPECTOR_PROFILER' }
+
+Environment variables used in profiled processes. FIB will be set to `40` on
```

Comment:
We probably don't really need to document the value of FIB since it's just empirical..also it may worth mention that it works with `test/fixtures/workload/fibonacci.js`.

Reply:
Would it make sense to remove FIB from the module and set it in the fixture instead?

Reply:
I went ahead and did that, but it's in a separate commit so I can pull it out if you think it's the wrong way to go.